chef/data_bags/crowbar/bc-template-ceph.schema

```diff
@@ -18,7 +18,9 @@
               "type": "map",
               "required": true,
               "mapping": {
-                "fsid": { "type": "str", "required": true }
+                "fsid": { "type": "str", "required": true },
+                "osds_in_total": { "type": "int", "required": true },
+                "replicas_number": { "type": "str", "required": true }
```

Comment:
That's related to the handling of the forms. The value of the select is only a string and not an integer. If we set it to ```int``` the validation fails.